app/components/sy-datepicker/component.js

```diff
@@ -70,7 +73,18 @@ export default class SyDatepicker extends Component {
     if (valid) {
       const parsed = parse(value);
 
-      return this["on-change"](parsed && parsed.isValid() ? parsed : null);
+      return this.args.onChange(parsed && parsed.isValid() ? parsed : null);
     }
   }
+
+  @action
+  updateCenter({ moment }) {
+    return (this.center = moment);
```

Comment:
Why is here a `return`?

Reply:
Don't know. Makes no sense.

Reply:
If the return value is needed i would vote for 
```js
this.center = moment
return this.center
```
otherwise delete it.

Reply:
`deleted`